_clients/data-prepper/data-prepper-reference.md

```diff
@@ -19,8 +19,11 @@ keyStorePassword | No | String | Password for keystore. Optional, defaults to em
 privateKeyPassword | No | String | Password for private key within keystore. Optional, defaults to empty string.
 serverPort | No | Integer | Port number to use for server APIs. Defaults to 4900
 metricRegistries | No | List | Metrics registries for publishing the generated metrics. Currently supports Prometheus and CloudWatch. Defaults to Prometheus.
+metricTags | No | Map | Key-value pairs as common metric tags to metric registries. The maximum number of pairs is 3. Note that `serviceName` is a reserved tag key with `DataPrepper` as default tag value. Its value could also be set through the environment variable `DATAPREPPER_SERVICE_NAME`. If `serviceName` is defined in `metricTags`, the value will overwrite those set through the above methods.
```

Comment:
> Its value could also be set through the

I recommend using active voice.

> Alternatively, administrators can set this value through the

Reply:
Thanks! These has been addressed in [ca1e7da](https://github.com/opensearch-project/documentation-website/pull/1469/commits/ca1e7da0795d2621ec2dd11045155b545cc8fa99).